gitlab/resource_gitlab_project_test.go

```diff
@@ -1343,3 +1379,15 @@ resource "gitlab_project" "foo" {
 }
 	`, rInt, rInt)
 }
+
+func testAccGitlabProjectConfigArchiveOnDestroy(rInt int) string {
+	return fmt.Sprintf(`
+resource "gitlab_project" "foo" {
+  name = "foo-%d"
+  path = "foo.%d"
+  description = "Terraform acceptance tests"
+  archive_on_destroy = true
+  archived = false
```

Comment:
Hmm in all our other tests we add this config to the `gitlab_project` resource. I'm not entirely sure it is still correct, but for consistency...

```suggestion
  archived = false

  # So that acceptance tests can be run in a gitlab organization with no billing.
  visibility_level = "public"
```

Reply:
Actually most of the resources in the `resource_gitlab_project_test.go` didn't set that attribute. However, I've pushed a commit on top this branch changing that 🎉